src/main/resources/META-INF/CustomMainMenu_At.cfg

```diff
@@ -18,7 +18,6 @@ protected net.minecraft.client.gui.GuiMainMenu func_73972_b(II)V #addDemoButtons
 protected net.minecraft.client.gui.GuiMainMenu func_73969_a(II)V #addDemoButtons
 protected net.minecraft.client.gui.GuiMainMenu func_73971_c(IIF)V #renderSkybox
 
-
 #public net.minecraft.client.renderer.entity.RenderPlayer func_96449_a(Lnet/minecraft/entity/EntityLivingBase;DDDLjava/lang/String;FD)V #renderLivingLabel
 #public net.minecraft.util.ResourceLocation field_110626_a #resourceDomain
-#public net.minecraft.util.ResourceLocation field_110625_b #resourcePath
+#public net.minecraft.util.ResourceLocation field_110625_b #resourcePath
```

Comment:
You can delete this file, the access transformer seems not to be needed at all...

Reply:
Fixed